contracts/utils/cryptography/ECDSA.sol

```diff
@@ -180,10 +180,14 @@ library ECDSA {
      *
      * See {recover}.
      */
-    function toEthSignedMessageHash(bytes32 hash) internal pure returns (bytes32) {
-        // 32 is the length in bytes of hash,
```

Comment:
Why remove this line, but not the one bellow.

Reply:
In the first commit, I used
```
mstore(0, 0x19457468657265756d205369676e6564204d6573736167653a0a333200000000)
```
instead of
```
mstore(0, "\x19Ethereum Signed Message:\n32")
```

Since there was no "\n32", I thought that line should be removed.
I think I should bring it back, thanks!